src/lib.rs

```diff
@@ -28,7 +28,8 @@ fn nro_main(nro: &NroInfo<'_>) {
             training::shield::handle_sub_guard_cont,
             training::directional_influence::handle_correct_damage_vector_common,
             training::sdi::process_hit_stop_delay,
-            training::tech::handle_change_status
+            training::tech::handle_change_status,
+            training::ledge::is_enable_transition_term_replace,
```

Comment:
You cannot put this hook here, it will cause unexpected crashes. The function you are hooking is not a `lua2cpp_common.nro` function, which is what these hooks are. `main` functions should be hooked elsewhere, like in the `training/mod.rs` main function.